config/pipeline.yaml

```diff
@@ -187,6 +202,8 @@ jobs:
   baseline-arm64-broonie: *baseline-job
   baseline-armel: *baseline-job
   baseline-x86: *baseline-job
+  baseline-x86-baylibre: *baseline-job
+  baseline-armel-baylibre: *baseline-job
```

Comment:
Please move this one between `baseline-armel` and `baseline-x86`

Reply:
Done.